lib/puppet/ssl/host.rb

```diff
@@ -22,8 +22,8 @@ class Puppet::SSL::Host
     The indirection key is the certificate CN (generally a hostname).
 DOC
 
-  attr_reader :name
-  attr_accessor :ca
+  attr_reader :name, :crl_path
+  attr_accessor :ca, :crl_usage
```

Comment:
I think it might make the code more readable to include boolean methods that it clear what you're asking at the point o fuse. Like
```ruby
def use_crl?
  !!@crl_usage
end

def use_crl_chain?
    @crl_usage == true || @crl_usage == :leaf
end
```